docs/admin_api/shutdown_room.md

```diff
@@ -79,13 +79,20 @@ Response:
 the structure can and does change without notice.
 
 First, it's important to understand that a room shutdown is very destructive. Undoing a shutdown is not as simple as pretending it
-never happened - work has to be done to move forward instead of resetting the past.
+never happened - work has to be done to move forward instead of resetting the past. In fact, in some cases it might not be possible
+to recover at all:
 
-1. For safety reasons, it is recommended to shut down Synapse prior to continuing.
+* If the room was invite only, your users will need to be re-invited.
+* If the room became empty, it'll be impossible to rejoin.
+* If the room was purged from your server, it may be inaccessible.
```

Comment:
I don't think this makes any difference?

Reply:
fair